channels.go

```diff
@@ -39,14 +39,12 @@ type Channel struct {
 	Creator            string         `json:"creator"`
 	IsArchived         bool           `json:"is_archived"`
 	IsGeneral          bool           `json:"is_general"`
-	IsGroup            bool           `json:"is_group"`
-	IsStarred          bool           `json:"is_starred"`
```

Comment:
That's definitely an inconsistency in their API then. I'd almost think of the websockets info payload as a separate use case (with potentially a separate struct).